functions.photon.php

```diff
@@ -90,23 +79,14 @@ function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
 		$args = rawurlencode_deep( $args );
 	}
 
-	/* Don't photon-ize WPCOM hosted images with only the following url args:
-	 *  `w`, `h`, `fit`, `crop`, `zoom`, `strip`, `resize`, `quality`
-	 * These args can just be added to the wpcom-version of the image, and save on latency.
-	 */
-	$is_wpcom_image_with_safe_args = false;
-	$allowed_wpcom_keys = array(
-		'w'       => true,
-		'h'       => true,
-		'fit'     => true,
-		'crop'    => true,
-		'zoom'    => true,
-		'strip'   => true,
-		'resize'  => true,
-		'quality' => true,
-	);
-	if ( wp_endswith( strtolower( $image_url_parts['host'] ), '.files.wordpress.com' ) && ! array_diff_key( $args, $allowed_wpcom_keys ) ) {
-		$is_wpcom_image_with_safe_args = true;
+	// Don't photon-ize WPCOM hosted images -- we can serve them up from wpcom directly.
```

Comment:
I think this is another case where the Jetpack code is more accurate.

Reply:
This was reverted specifically by @darthhexx in r160617-wpcom, _after_ we added it to Jetpack in #6497.

@darthhexx, considering this file is shared with wpcom, what do you think we should do?

Reply:
They need to be removed, because Photon and WPCOM are feature compatible. Therefore there is no need for a curated list of supported parameters.